lib/server.js

```diff
@@ -43,13 +43,14 @@ module.exports = {
   registerRoutes (app, server) {
     _.each(app.routes, route => server.route(route))
 
-    if (app.config.web.assets) {
+    if (app.config.main.paths.www) {
```

Comment:
In express I log a message when `app.config.main.paths.www` is not set. Like this user know it have to set it if he want static assets :) look https://github.com/trailsjs/trailpack-express4/blob/master/lib/server.js#L37 it also have to be on documentation but I think it's good to say it here